### ISSUE TITLE
Recipe quirks

### ISSUE THREAD

**Issue by Cylkin** [reteo/Survival-Industry#119]
(EnderStorage) Ender Chests can still be crafted using regular ender pearls instead of singularities, the recipes can be found with the "usage" function in NEI on ender pearls. Only the white wool recipe is gone. Same applies to Ender Tanks and probably the bags (untested).

Packed Ice has a difficult recipe using the Compactor in -80C as well as a simple 2x2 crafting recipe using regular ice. The latter halves the ice requirements for the vegan ender pearl recipe without adding any complexity to the process. This has hardly any impact on progression, but as this pack generally adheres to the RoC progression and there seem to be no side effects in removing the simpler one, this might be worth looking into for completeness' sake.

Tested with a fresh install of 1.6.1 (share code p3cKsOQW).

**Comment by reteo** [comment]
Good catch on both!  Since they aren't critical issues (we'll consider Ender Chests a time-limited offer), and RL seems to have me by the goolies, I'll probably be a short while in correcting them.
